### PR TITLE
Add StreetView button tooltip

### PR DESCRIPTION
**Pull request by geographika**

### CODE REVIEW

**Review by chrismayer — APPROVED** [review]

File: app/view/button/StreetViewTool.js [hunk 1]
```diff
@@ -33,7 +33,14 @@ Ext.define('CpsiMapview.view.button.StreetViewTool', {
     name: 'streetViewToolButton',
 
     /**
-     * Enable toogle mode for the button.
+     * The tooltip to display when hovering over the button
+     *
+     * @private
+     */
+    tooltip: 'Activate the StreeView tool',
```
Comment (marcjansen) [inline app/view/button/StreetViewTool.js:40]: Typo: Stree**t**

**Review by marcjansen — APPROVED** [review]

File: app/view/button/StreetViewTool.js (hunk 1 above)
Comment (chrismayer) [reply app/view/button/StreetViewTool.js:40]: Good catch, was too early for me, I guess :wink:

Comment (geographika) [reply app/view/button/StreetViewTool.js:40]: Was too late at night for me! Thanks!

Comment (marcjansen) [reply app/view/button/StreetViewTool.js]: 😅

**Pull request merged**